### PR TITLE
New recipe for mauveAligner

### PR DESCRIPTION
**Pull request by jamesabbott**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

This is a component of Mauve, providing the mauveAligner and progressiveMauve tools

Mauve is a rather complex build which I have working (manually) under linux, however Clang++ does not like some definitions so osx builds are disabled for now. I'll try to sort these out once the whole linux build is working

### CODE REVIEW

File: recipes/mauvealigner/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  url: https://depot.galaxyproject.org/software/mauveAligner/mauveAligner_1.2.0_src_all.zip
+  sha256: 0d0cf5dd560cf062d4bfb01b121b38865f3a7fae6312d4169eb696add82cefc7
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - autoconf
+    - automake
+    - libtool
+    - libgenome
```
Comment (bgruening) [inline recipes/mauvealigner/meta.yaml]: libgenome and the others below needs to be in the host section, I think

Comment (jamesabbott) [reply recipes/mauvealigner/meta.yaml]: ooo...thanks...Is this a new high-speed linting service?

File: recipes/mauvealigner/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  skip: True [osx]
+  number: 0
+
+source:
+  url: https://depot.galaxyproject.org/software/mauveAligner/mauveAligner_1.2.0_src_all.zip
```
[the remark this answers is not in the document]
Comment (jamesabbott) [reply recipes/mauvealigner/meta.yaml]: 2.4.0 is buggy with stability problems, which are addressed in svn but have never  made it to a release, The svn snapshots I pushed to cargo-port should have these issues resolved...

**Comment by jamesabbott** [comment]
PR entangled with other recipes...#9579 opened on clean branch

**Pull request closed without merging**